README.md

```diff
@@ -83,12 +83,15 @@ This is an experimental web client for Jellyfin based on Vue.js. We welcome all
 ### Running a production build
 
 Instead of a development version, you can run a server with a production-ready build of the client
-directly with Nuxt, so you can verify in advance how the client will work in your production environment:
+directly with Nuxt, so you can verify in advance how the client will work in your production environment with a reverse proxy:
```

Comment:
Not even a RP, you can just use a classic HTTP server or what else. I'd just add a different sentence like "Please note this isn't suitable for a real production hosting." or the same sentence they put in the Nuxt doc if you wish :)